solarized-definitions.el

```diff
@@ -252,7 +252,7 @@ the \"Gen RGB\" column in solarized-definitions.el to improve them further."
                  (,@fg-base1 ,@bg-base02 ,@fmt-revbb :box nil))
                 (mode-line-inactive    ; StatusLineNC
                  (,@fg-base00 ,@bg-base02 ,@fmt-revbb :box nil))
-                (region (,@fg-base01 ,@bg-base03 ,@fmt-revbb)) ; Visual
+                (region (:background base02 :foreground nil :inverse-video nil))
```

Comment:
This comes directly from the Vim theme, so I’m not likely to change it. But soon I’ll merge something like #191 which would make it easy to define variations like this locally (while still being able to refer to `base02`, etc.).